### ISSUE TITLE
Reordering of data input side links

### ISSUE THREAD

**Issue by karindalziel** [CDRH/african_poetics#131]
Lorna please fill in

**Comment by jduss4** [comment]
Lorna is putting this list together

**Comment by DawesL2** [comment]
Here is the list: 	

**Add New or Edit**
News Items


**Edit Records & Connections**
Educations
Events
People
Relationships
Works

**Edit Related Roles**
News Item Roles
Work Roles

**Reference Info**
             No change

**Comment by jduss4** [comment]
@DawesL2 I have made the changes and put them on production.  I only deviated twice from your instructions, let me know what you think about them:

- you didn't say where you wanted the "meta commentaries," which had been in the same place as News Item Roles and Work Roles, so I left it there and just renamed it "Related Commentaries" to try to make it clear that it is for commentaries which need to be linked to other commentaries.  Let me know if you want me to move it somewhere else.

- I added "news item content types" to the "reference info" section, since I think we must have forgotten to add that table to the list when we made it a few months ago
